src/tailwind.js

```diff
@@ -49,27 +43,24 @@ module.exports = {
         },
         // END - redesign
         inherit: {
-          default: "inherit",
-        },
-        black: {
-          default: "#000000",
+          DEFAULT: "inherit",
         },
         blue: {
-          default: "#0099cc",
+          DEFAULT: "#0099cc",
           300: "#f3f8fc",
```

Comment:
should we drop the `DEFAULT`? and in place, the mental model is to use 500 as "default"?